src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -420,6 +420,11 @@ private StringBuilder BuildBreakInsert(string condition)
             return await _debugger.CmdAsync(cmd.ToString(), resultClass);
         }
 
+        public virtual Task<Results> BreakWatch(string address, uint size, ResultClass resultClass = ResultClass.done)
+        {
+            throw new NotImplementedException();
```

Comment:
do we want to put it inside the gdb specific portion of the command factory given that clrdbg also can't support databps?